dotCMS/src/main/java/com/dotcms/publishing/PublisherAPI.java

```diff
@@ -58,6 +58,12 @@
      */
     List<FilterDescriptor> getFiltersDescriptorsByRole(final User user) throws DotDataException;
 
+    /**
+     * Returns true if the filter descriptor exists
+     * @return boolean
+     */
+    boolean existsFilterDescriptor(final String filterKey);
```

Comment:
Probably it's good idea to rename it to `filterDescriptorExists`

Reply:
Agree